Integrations/MimecastV2/MimecastV2.py

```diff
@@ -1479,6 +1471,605 @@ def download_attachment_request(attachment_id):
     return response.content
 
 
+def find_groups():
+    api_response = create_find_groups_request()
+
+    markdown_output = find_groups_api_response_to_markdown(api_response)
+    entry_context = find_groups_api_response_to_context(api_response)
+
+    return_outputs(markdown_output, entry_context, api_response)
+
+
+def create_find_groups_request():
+    api_endpoint = '/api/directory/find-groups'
+    query_string = demisto.args().get('query_string', '').encode('utf-8')
+    query_source = demisto.args().get('query_source', '').encode('utf-8')
+    pagination = demisto.args().get('pagination')
```

Comment:
not a good name(change in the yml as well)

Reply:
Done.